src/main/java/commands/AddCommand.java

```diff
@@ -0,0 +1,58 @@
+package commands;
+
+import exceptions.SnomException;
+import storage.Storage;
+import tasks.*;
+import ui.Snomio;
+
+public class AddCommand extends Command{
+    public AddCommand(CommandEnum commandType, String content) {
+        super(commandType, content);
+    }
+
+    /**
```

Comment:
Concise and clear documentation, really helped me to understand the functionality of this method. Good job:D

Reply:
Thanks!